src/languageProvider/locales/en_US.json

```diff
@@ -87,26 +87,35 @@
   "sort.asc": "Ascending",
   "sort.label": "Sort By",
   "sort.desc": "Descending",
+  "str.amount": "Amount",
   "str.bot": "BOT",
+  "str.close": "Close",
+  "str.copy": "Copy",
   "str.end": "Ended",
+  "str.from": "From",
+  "str.to": "To",
   "str.outcome": "OUTCOMES",
   "str.qtum": "QTUM",
   "str.raise": "Raised",
   "str.RAISE": "RAISED",
   "str.sort": "Sort",
+  "str.transactionId": "Transaction ID",
   "topbar.betting": "Betting",
   "topbar.completed": "Completed",
   "topbar.create": "Create an Event",
   "topbar.event": "Event",
   "topbar.events": "Events",
   "topbar.setting": "Setting",
   "topbar.voting": "Voting",
+  "transactionSentDialog.successMsg": "Success",
+  "transactionSentDialog.failureMsg": "Oops, something went wrong",
+  "transactionSentDialog.waitingMsg": "Transaction sent. Waiting for confirmations.",
   "walletHistory.transactionHistory": "Transaction History",
   "walletHistory.time": "Time",
-  "walletHistory.from": "From",
```

Comment:
Can we keep a list of missing translation somewhere?

Reply:
this wasn't deleted, just moved from `walletHistory.from` to `str.from`. we've been using `str.xxx` for generic strings that can be used in more than one place.